input_files/strtx_III_dsp/process_specific_params.txt

```diff
@@ -29,10 +29,10 @@ standard_libs="/CMC/kits/tsmc_65nm_libs/tcbn65gplus/TSMCHOME/digital/Front_End/t
 worst_case_libs="/CMC/kits/tsmc_65nm_libs/tcbn65gplus/TSMCHOME/digital/Front_End/timing_power_noise/NLDM/tcbn65gplus_140b/tcbn65gpluswc.lib /CMC/kits/tsmc_65nm_libs/tpzn65gpgv2/TSMCHOME/digital/Front_End/timing_power_noise/NLDM/tpzn65gpgv2_140c/tpzn65gpgv2wc.lib"
 
 
-#metal_layers=8
-metal_layers=7
+metal_layers=8
```

Comment:
Do you mean to use 8 metal layers or 5?  It looks like it is set to 8 but then overwritten to 5.

Reply:
@vaughnbetz  the configuration files currently support doing multiple runs (ie sweeping parameters) by putting another value into the same parameter (its very confusing and I will change it when I can) to be honest I would like to switch the entire config files into yaml instead of text file

Reply:
OK, in that case just comment that it does that.